.vscode/tasks.json

```diff
@@ -2,11 +2,49 @@
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
   "version": "2.0.0",
+  "presentation": {
+    "echo": false,
+    "reveal": "always",
+    "focus": false,
+    "panel": "dedicated",
+    "showReuseMessage": true
+  },
   "tasks": [
```

Comment:
btw, i really like this format for our own tasks (the whole dependsOn stuff is nice)